bugbug/commit_features.py

```diff
@@ -808,34 +809,47 @@ def __call__(self, commit, **kwargs):
 
 
 class Files(object):
+    name = "files"
+
+    def __call__(self, commit, **kwargs):
+        return commit["files"]
+
+
+def _pass_through_tokenizer(doc):
+    return doc
+
+
+class FrequentFilesVectorizer(BaseEstimator, TransformerMixin):
     def __init__(self, min_freq=0.0014):
         self.min_freq = min_freq
+        self.vectorizer = CountVectorizer(
+            tokenizer=_pass_through_tokenizer, lowercase=False
```

Comment:
Nit: use a lambda so you don't have to define `_pass_through_tokenizer`

Reply:
Passing a local function (including lambda) cannot be pickled.